environment.js

```diff
@@ -1,2 +1,4 @@
-export const BATCH_SIZE = process.env.BATCH_SIZE  ? process.env.BATCH_SIZE  : 50;
-export const EMAIL_FROM = process.env.EMAIL_FROM  ? process.env.EMAIL_FROM  : 'noreply@pers-berichten.vlaanderen.be';
+export const BATCH_SIZE = process.env.BATCH_SIZE  ? parseInt(process.env.BATCH_SIZE)  : 50;
```

Comment:
Ter info: je kan dit korter schrijven in javascript als `parseInt(process.env.BATCH_SIZE || 50)` (zelfde voor de andere variabelen)